feat(core): Add $uuid() global helper for expressions

kurucaner opened the pull request:
- Add $uuid helper that generates a new UUID v4 on each access
- Import uuid v4 function in WorkflowDataProxy
- Add test to verify $uuid returns valid UUID format
- Allows users to generate unique IDs in expressions like {{ $uuid }}

## Summary

This PR adds a new `$uuid` global helper to n8n's expression system, allowing users to generate unique UUID v4 identifiers directly in their workflow expressions.

### Changes
- Added `$uuid` helper to the WorkflowDataProxy class that returns a new UUID v4 each time it's accessed
- Imported `uuid` v4 function in `workflow-data-proxy.ts`
- Added unit test in `workflow-data-proxy.test.ts` to verify the helper returns valid UUID format

### Usage
Users can now use `{{ $uuid }}` in their expressions to generate unique identifiers:
// Example: Generate a unique request ID
{{ $uuid }}
// Output: "a3bb189e-8bf9-4a1e-9e5c-6d6e4c3f2a1b"

### Testing
- Run the workflow package tests: `cd packages/workflow && pnpm test workflow-data-proxy.test.ts`
- Test validates UUID format using regex pattern for UUID v4

kurucaner committed:
feat(workflow): add  global helper for expressions

- Add  helper that generates a new UUID v4 on each access
- Import uuid v4 function in WorkflowDataProxy
- Add test to verify  returns valid UUID format
- Allows users to generate unique IDs in expressions like {{  }}

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/n8n-io/n8n?pullRequest=23595) <br/>All committers have signed the CLA.

kurucaner committed:
feat(core): Add $uuid global helper for expressions

- Add $uuid helper that generates a new UUID v4 on each access
- Import uuid v4 function in WorkflowDataProxy
- Add test to verify $uuid returns valid UUID format
- Add autocomplete support in Code node and expression editors
- Add i18n translations for tooltip documentation
- Allows users to generate unique IDs in expressions like {{ $uuid }}

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master

kurucaner committed:
Merge branch 'master' into master
